website_and_docs/content/documentation/support_packages/chrome_devtools.en.md

```diff
@@ -5,16 +5,21 @@ weight: 5
 aliases: ["/documentation/en/support_packages/chrome_devtools/"]
 ---
 
-Selenium 4 alpha versions have much awaited native support for Chrome DevTools 
-Protocol through "DevTools" interface. This helps us getting Chrome Development 
-properties such as Application Cache, Fetch, Network, Performance, Profiler, 
-Resource Timing, Security and Target CDP domains etc.
-
-Chrome DevTools is a set of web developer tools built directly into the Google 
-Chrome browser. DevTools can help you edit pages on-the-fly and diagnose 
-problems quickly, which ultimately helps you build better websites, faster.
-
-## Emulate Geo Location:
+{{% pageinfo color="warning" %}}
+<p class="lead">
+  While Selenium 4 provides direct access to Chrome DevTools Protocol, it is highly encouraged that
```

Comment:
"the Chrome DevTools Protocol (CDP)"

There are two changes here: first "the" which reads more naturally. Secondly, we're providing the abbreviation next to the definition, so it's okay for you to use "CDP" later in this explanation.